samples/remote_host_attestation/host.c

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host_verify.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+size_t get_filesize(FILE* fp)
+{
+    size_t size = 0;
+    fseek(fp, 0, SEEK_END);
+    size = (size_t)ftell(fp);
+    fseek(fp, 0, SEEK_SET);
+
+    return size;
+}
+
+bool read_binary_file(
+    const char* filename,
+    uint8_t** data_ptr,
+    size_t* size_ptr)
+{
+#ifdef _WIN32
+#pragma warning(disable : 4996)
+#pragma warning(push)
+#endif
```

Comment:
Are samples built with /WX? Since samples are meant to be illustrative, I would probably remove these checks if they aren't needed to enhance clarity.

Reply:
No, they aren't; I saw the warnings on Windows but they did not prevent from building the sample either. I will remove them later.